generic/tests/cfg/iozone_windows.cfg

```diff
@@ -29,3 +29,13 @@
             force_create_image_stg1 = yes
             remove_image_stg0 = yes
             remove_image_stg1 = yes
+        - gpt_disk_test:
```

Comment:
How about the case name `with_gpt_disk`? the word "test" covers the case `iozone_windows`.

Reply:
that's ok, will update, thanks~